src/Frontend/store/handler/handleLogin.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import Locale from '../../util/http/LocaleService';
+import counterpart from 'counterpart';
+
+/**
+ * Handler which updates the locale.
+ *
+ * @returns {Object} The updated locale.
+ */
+export default () => {
+  const actual = Locale.getLocale();
+  if (actual !== counterpart.getLocale()) {
+    counterpart.setLocale(actual);
+  }
+  return { locale: actual };
```

Comment:
rename ``actual`` to ``locale``, so the statement could be ``{ locale }`` which is much more readable.